src/site/content/en/blog/webxr-pt1/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Virtual reality comes to the web
+subhead: A few basicis to prepare you for a spectrum of immersives experiences&#58; virtual reality, augmented reality, and everything in between.
+authors:
+  - joemedley
+date: 2019-10-30
+hero: hero.jpg
+alt: A person using a virtual reality headset.
+description:
+  Virtual reality has come to the web in Chrome 79. Based on the WebXR Device
+  API, this launch is the foundation for immersive features to come later such
+  as augmented reality. Other browsers will be supporting the spec soon,
+  including [update before launch].
+tags:
+  - post
+  - ar
+  - augmented-reality
+  - virtual-reality
+  - vr
```

Comment:
In a book index, which shares underlying theory with tags, one of these would have 'see the other` instead of a page number. Synonyms are a must for indexes and tags because it's a way to account for a reader having a different term in their head than what's used in the article. 

Speaking of that, We really wouldn't be following the principle correctly if we didn't implement the web's version of a 'see' reference. Would you like me to open a feature request for tag redirects?

Reply:
I just created #1808 for the tag redirect feature request. I will support tag synonyms once we have the tag redirect infrastructure in place. Once that feature is implemented you can add synonyms wherever you like. If we're start using synonyms now, without a redirect infrastructure, we introduce the possibility of half our content being tagged `virtual-reality` and the other half being tagged `vr`. The content review process is already lengthy when you follow the content checklist, so I'm hesitant to add yet another manual check right now. Furthermore not everyone is following the content checklist rigorously, so there is a very real possibility that some reviewers will forget to check if all relevant tags have been applied to an article.